dist/src/main/package/www/languages/mc_locale_gui-en_us.json

```diff
@@ -415,6 +415,7 @@
     "PHYSICAL_MEMORY": "Physical memory (RAM)",
     "PLIVO": "Plivo",
     "POLL_FREQUENCY_MINUTES": "Poll frequency (minutes)",
+    "AUTORIZED_USER":"Autorized user",
```

Comment:
@Fraid No need to touch `mc_locale_gui-*.json` files. We have to add our changes only on `mc_locale_gui_source_en.json` file